src/sagemaker/jumpstart/accessors.py

```diff
@@ -307,6 +299,21 @@ def get_model_specs(
                 model_specs.set_hub_content_type(HubContentType.MODEL_REFERENCE)
                 return model_specs
 
+            except Exception as ex:
+                logging.info(
```

Comment:
warn?

Reply:
imo info is good enough, its just how we try to access the model or model ref with as minimum input as possible. The purpose here is to just provide context to the user that this is what we are doing to get the information they are asking for. Why do you think this should be warn?